app/views/admin/shared/_nested_sidebar.html.erb

```diff
@@ -0,0 +1,29 @@
+<% menu_items.each do |group_name, group| %>
```

Comment:
I renamed the file from _nested_sidebar.erb to _nested_sidebar.html.erb. and added some classes for current links and making the dropdowns expandable and collapsible.

Reply:
Great change!